configs/config_standalone.json

```diff
@@ -141,5 +141,48 @@
         }
       }
     ]
+  },
+  "dynamic_resources": {
```

Comment:
Could u also modify other json files? The other json files are still causing startup errors. See https://github.com/mosn/layotto/actions/runs/3369341972/jobs/5588886336

![image](https://user-images.githubusercontent.com/26001097/199273283-90c08b43-d9c5-4cbe-a417-06dd7d04b7f7.png)